bin/ka-clone

```diff
@@ -520,6 +553,9 @@ def _cli_process_current_dir(cli_args):
 
     install_global_git_hooks()
 
+    if is_webapp_repo():
+        run_webapp_make_hooks()
```

Comment:
I think it would be cleaner to allow _any_ repo to have a `make hooks` rule, and if it exists to run it as part of the ka-clone process.  Then, instead of `is_webapp_repo`, you'd do something like:
```
if os.path.exists(os.path.join(repo_root, "Makefile")) and "\nhooks: " in open(os.path.join(repo_root, "Makefile")).read():
    run_make_hooks()
```

Reply:
ok